src/assets/src/components/meetingTables.tsx

```diff
@@ -138,7 +145,22 @@ function UnstartedMeetingEditor (props: UnstartedMeetingEditorProps) {
 }
 
 function StartedMeetingEditor (props: MeetingEditorProps) {
-    const attendee = props.meeting.attendees[0];
+    const attendee = props.meeting.attendees[0] ?? null;
+    if (!attendee) return (
```

Comment:
line 148-163 is identical to line 84-98. Could reformat to use one single source.

Reply:
I went ahead and made the "Invalid Meeting" component generic